user_manual/changes/T406-026.yaml

```diff
@@ -0,0 +1,42 @@
+type: new-feature
+title: Add ``P_Shapes`` property
+description: |
+    This property, when called on the ComponentList of a record definition,
+    will return the list of all possible shapes that a record value of this
+    type can take. Consider the following record definition:
+
+	.. code::
```

Comment:
Looks like you mistakenly used tabs here

Reply:
Indeed, thanks!